src/App.js

```diff
@@ -30,9 +30,9 @@ export default function App() {
         <Route path="/edit-user">
           <UserProfileForm onSubmit={addUser} />
         </Route>
-        <Route path="/favorites"></Route>
+        <Route path="/favorites">Coming soon</Route>
         <Route path="/create">
-          <CreateContactForm onSubmit={addContact} />
+          <CreateContact onSubmit={addContact} />
```

Comment:
Cant comment on line 43, but I would move your homepage to the top and add an exact to the path. Makes it easier to read the code in my opinion.